src/main/java/net/raphimc/viabedrock/protocol/rewriter/ResourcePackRewriter.java

```diff
@@ -35,7 +35,12 @@ public static ResourcePack.Content bedrockToJava(final ResourcePacksStorage reso
         final ResourcePack.Content javaContent = new ResourcePack.Content();
 
         resourcePacksStorage.iterateResourcePacksBottomToTop(pack -> {
-            convertGlyphSheets(pack.content(), javaContent);
+            try {
```

Comment:
Remove the try catch (If your code randomly throws exceptions, you should fix that by adding more sanity checks)